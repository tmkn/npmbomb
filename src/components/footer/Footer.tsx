/** @jsx jsx */
import { jsx, css } from "@emotion/core";

import { mq } from "../../css";

const footerStyle = css({
    display: "flex",
    fontFamily: "'Open Sans', sans-serif",
    fontSize: "0.8rem",
    justifyContent: "center",
    marginTop: "4rem",
    marginBottom: "2rem",
    color: "#616161"
});

const Footer: React.FC = () => {
    return (
        <div css={footerStyle}>
            <span>@tmkndev</span>
        </div>
    );
};

export default Footer;
