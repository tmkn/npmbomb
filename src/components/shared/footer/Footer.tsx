/** @jsx jsx */
import { jsx, css } from "@emotion/core";

import { mq, sansSerifFont, textColor, secondaryColor } from "../../../css";

const footerStyle = css({
    [mq[0]]: {
        display: "flex",
        fontFamily: `"${sansSerifFont}"`,
        fontSize: "0.8rem",
        justifyContent: "center",
        marginTop: "4rem",
        marginBottom: "2rem",
        color: `${textColor}`,
        "& a": {
            textDecoration: "underline",
            textDecorationColor: secondaryColor
        }
    }
});

const Footer: React.FC = () => {
    return (
        <div css={footerStyle}>
            <span>
                made with ♥️ by <a href="https://twitter.com/tmkndev">@tmkndev</a>
            </span>
        </div>
    );
};

export default Footer;
