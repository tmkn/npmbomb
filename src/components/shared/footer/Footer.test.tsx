import React from "react";
import { render, fireEvent } from "@testing-library/react";

import Footer from "./Footer";

describe("<Footer />", () => {
    test("matches snapshot", () => {
        const { asFragment } = render(<Footer />);

        expect(asFragment()).toMatchSnapshot();
    });
});
