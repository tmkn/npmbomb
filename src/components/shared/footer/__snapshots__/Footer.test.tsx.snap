// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-138a4w9-Footer"
  >
    <span>
      hecho con ♥️ por 
      <a
        class="css-1ka21sq"
        href="https://twitter.com/tmkndev"
      >
        @tmkndev
      </a>
    </span>
  </div>
</DocumentFragment>
`;
